protobuf/echo/protocol_buffer_echo.cmake

```diff
@@ -1,8 +1,34 @@
-function(protocol_buffer_echo_generator target input)
-    if (CMAKE_CROSSCOMPILING)
-        find_package(emil COMPONENTS Infra Hal Crypto Services Protobuf REQUIRED)
+function(emil_fetch_echo_plugins)
+    set(emil_version "3.0.0") # x-release-please-version
+
+    if (CMAKE_HOST_WIN32)
+        FetchContent_Declare(echoplugins
+            URL https://github.com/philips-software/amp-embedded-infra-lib/releases/download/v${emil_version}/emil-${emil_version}-win64.zip
```

Comment:
Can you extract a function which gives the download suffix for the current platform?

Reply:
I would even perhaps opt for allowing custom urls?
`EMIL_OPTIONAL_BINARY_RELEASE_URL`
If set it uses this url, else the default download location? (Thinking about how some projects would react to seeing a dependency om some open source tooling/hosting). I would assume they would rather use a local hosting solution instead.

Reply:
I don't think that will be necessary. We use a fully transparent and auditable release process. Only the release bot creates (signed) releases.

Next to this dependency we already have other dependencies to open-source projects.

I do still think that promoting the plug-in binary variables to internal cached variables will solve most of the issues.